anyway/apis/markers/api.py

```diff
@@ -1,8 +1,11 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
```

Comment:
Since we are intending the use for python 3.6 and above (3.7 and above?) this is useless

Reply:
Will be removed